prog/vm/github_runner.rb

```diff
@@ -138,13 +129,15 @@ def before_run
       github_runner.update(runner_id: response[:runner][:id], ready_at: Time.now)
       # ./env.sh sets some variables for runner to run properly
       vm.sshable.cmd("./env.sh")
-      vm.sshable.cmd("common/bin/daemonizer 'sudo -u runner /home/runner/run.sh --jitconfig #{response[:encoded_jit_config].shellescape}' runner-script")
+
+      command = "/bin/bash -c 'sudo -u runner ./run.sh --jitconfig #{response[:encoded_jit_config].shellescape}'"
+      vm.sshable.cmd("sudo systemd-run --working-directory '/home/runner' --unit #{SERVICE_NAME} --remain-after-exit -- #{command}")
```

Comment:
could you please test passing "--uid runner --gid runner" to systemd-run instead of "sudo -u runner"

Reply:
Yeah it works with  "--uid runner --gid runner"